contracts/tests/CounterTest.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+contract CounterTest {
+    address public ops;
+    uint256 public count;
+    uint256 public lastExecuted;
+
+    constructor(address payable _ops) {
+        ops = _ops;
+    }
+
+    // solhint-disable not-rely-on-time
+    function increaseCount(uint256 amount) external {
```

Comment:
so this one is the one we'll use for the ops-sdk hello world, that anyone can call ?